src/btree/bt_debug.c

```diff
@@ -657,6 +657,30 @@ __wt_debug_tree_all(
 	return (ret);
 }
 
+/*
+ * __wt_debug_tree_cursor --
+ *	Dump the in-memory information for a tree, including leaf pages
+ *	starting from a user's cursor and the root of the tree.
+ */
+int
+__wt_debug_tree_cursor(WT_CURSOR *cursor, const char *ofile)
+    WT_GCC_FUNC_ATTRIBUTE((visibility("default")))
+{
+	WT_CURSOR_BTREE *cbt;
+	WT_DECL_RET;
+	WT_SESSION_IMPL *session;
+
+	session = (WT_SESSION_IMPL *)cursor->session;
+	cbt = (WT_CURSOR_BTREE *)cursor;
+
+	/*
+	 * Dump only the cursor referenced page.
+	 */
+	WT_WITH_BTREE(session, cbt->btree, ret = __wt_debug_page(
+	    session, NULL, cbt->ref, ofile));
+	return (ret);
+}
+
```

Comment:
Isn't this the same as `wt_debug_cursor_page()`?

Reply:
Yes! After I made the change, I didn't realize the same thing was just a bit further down. I'll fix.